base/setup/reactos/lang/id-ID.rc

```diff
@@ -16,7 +16,7 @@ CAPTION "Penyetelan ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
     AUTORADIOBUTTON "Pasang ReactOS", IDC_INSTALL, 7, 7, 277, 10, WS_GROUP | WS_TABSTOP
-    LTEXT "Install a new copy of ReactOS. This option does not keep your files, settings and programs. You can make changes to disks and partitions.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
+    LTEXT "Pasang instalasi ReactOS yang baru. Ini tidak akan menjaga berkas, pengaturan, dan program Anda. Anda bisa membuat perubahan pada disk dan partisi.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
```

Comment:
```suggestion
    LTEXT "Pasang instalasi ReactOS yang baru. Ini tidak akan menjaga berkas, pengaturan, dan program Anda. Anda bisa membuat perubahan pada disk dan partisi lain.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
```

Reply:
for me the former looks better for me, why saying "other partitions" while you're installing a clean copy of ROS?

Reply:
it's the case when you install this setup while you have booted on the other OS. So, this setup sugest you to install a clean copy to other partition if you feel unsure to follow up this option. cmiiw

Reply:
If you leave this as you want to keep your commit. It's okay